pkgs/tools/security/sudo/default.nix

```diff
@@ -28,6 +28,7 @@ stdenv.mkDerivation rec {
     "--with-logpath=/var/log/sudo.log"
     "--with-iologdir=/var/log/sudo-io"
     "--with-sendmail=${sendmailPath}"
+    "--enable-tmpfiles.d=no"
```

Comment:
What's the reason for this?

Reply:
Indeed it could use a comment. I'll have to revert it to recall why this was necessary.

Reply:
Dropping-- presumably this is forcing the expect value so as to avoid performing a cross-unfriendly check.  Perl is on the path to sudo and so perhaps this should be revisited once perl works.